README.md

```diff
@@ -16,41 +19,48 @@ For tutorials, samples, quick starts and other documentation, visit [Azure for J
 ### Prerequisites
 Java 8 or later is required to use the November 2019 client libraries, otherwise Java 7 or later is required.
 
-## Packages available
+## Release branches (Release tagging)
+
+### Master branch
+The master branch represent more recent code with new features and bug fixes. It does **not** represent latest released **GA** SDK. See [below](#Client-GA-November-2019-Releases) for latest **GA** release.<br/>
+For each package we release there will be a unique git tag created that contains the name and the version of the package to mark the commit of the code that produced the package. This tag will be used for servicing via hotfix branches as well as debugging the code for a particular preview or stable release version.
+Format of the release tags are &lt;package-name&gt;_&lt;package-version&gt;. For more information please [see](https://github.com/Azure/azure-sdk/blob/master/docs/policies/repobranching.md#release-tagging)
```

Comment:
You can use backtick instead of less than and greater than if you want it to show as-is.

Reply:
Also the reason why it shows up as italics. Consider using `<package-name>_<package-version>` instead. (with backticks)

Reply:
It does not show as italics when we see MD file in browser. Try to open this in browser https://github.com/Azure/azure-sdk-for-java/blob/176a3504db0b0071186df1b6b2f2bd6a1369ed23/README.md#master-branch